lib/rotp/totp.rb

```diff
@@ -47,6 +47,30 @@ def verify_with_drift(otp, drift, time = Time.now)
       times.any? { |ti| verify(otp, ti) }
     end
 
+    # Verifies the OTP passed in against the current time OTP
+    # and adjacent intervals up to +drift+.  Excludes OTPs
+    # from prior_time and earlier.  Returns time value of
+    # matching OTP code for use in subsequent call.
+    # @param [String] otp the OTP to check against
+    # @param [Integer] drift the number of seconds that the client
+    #     and server are allowed to drift apart
+    # @param [Integer] time value of previous match
+    def verify_with_drift_and_prior(otp, drift, prior_time = nil, time = Time.now)
+      # calculate normalized bin start times based on drift
```

Comment:
Rather than two completely separate functions would it make sense to implement `verify_with_drift` in terms of `verify_with_drift_and_prior`,  or simply to make `prior_time` and additional optional argument to `verify_with_drift`?